src/main/java/net/mcreator/ui/init/BlocklySpecialFilesLoader.java

```diff
@@ -19,37 +19,48 @@
 
 package net.mcreator.ui.init;
 
+import net.mcreator.blockly.data.ExternalBlockLoader;
 import net.mcreator.io.FileIO;
 import net.mcreator.plugin.PluginLoader;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
 import java.util.regex.Pattern;
 
-public class BlocklyJavaScriptsLoader {
+public class BlocklySpecialFilesLoader {

```

Comment:
I would also keep BlocklyJavaScriptsLoader and toolbox loader separate as they are different things, together with different folders